src/math/exp_fcn_hifi.c

```diff
@@ -278,4 +284,130 @@ int32_t sofm_exp_int32(int32_t x)
 
 	return AE_MOVAD32_L(AE_MOVINT32X2_FROMINT64(ts));
 }
+
+/* Xtensa macros */
+/* Convert a float number to fractional Qnx.ny format. Note that there is no
+ * check for nx+ny number of bits to fit the word length of int. The parameter
+ * qy must be 31 or less.
+ */
+static inline int sofm_q_CONVERT_FLOAT(float a, int b)
```

Comment:
sorry, my mistake - I did not see these were in the hifi file and not public APIs. These can be renamed to ```exp_hifi_q_convert_float()``` all lower case and so on.